src/ReviewPanel.jsx

```diff
@@ -1,31 +1,36 @@
-import React, {Component} from 'react'
+import React, {PropTypes} from 'react'
 import { Panel, Row, Col } from 'react-bootstrap'
 
-export default class ReviewPanel extends Component {
-  render() {
-    const time = this.props.timestamp;
-    const good = this.props.good;
-    const reviewer = this.props.reviewer;
-    const header = reviewer + " on " + new Date(time).toDateString();
-    let flags = null;
-    if (good) {
-      flags = (
-        <p style={{color: "#7CFC00"}}>+</p>
-      )
-    } else {
-      flags = (
-        <p style={{color: "#DC143C"}}>-</p>
-      )
-    }
-    const body=this.props.children;
-
-    return (
-      <Panel header={header} key={this.props.timestamp}>
-        <Row>
-          <Col sm={1}>{flags}</Col>
-          <Col>{body}</Col>
-        </Row>
-      </Panel>
+const ReviewPanel = ({review}) => {
+  if(!review) return <div></div>;
+  const time = review.timestamp;
+  const good = review.positive;
+  const reviewer = review.team_member_name;
+  const header = reviewer + " on " + new Date(time).toDateString();
```

Comment:
Is there a call for that in JS? I couldn't find anything.

Also I would argue that in this limited scope, short and sweet names trump long-winded names.